### PR TITLE
setup 

### PR DESCRIPTION
**Pull request by ybadr99**
- setup react app
- setup linters configuration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,117 @@
...
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> I would like to thank [Leaderboard API](https://microverse.notion.site/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3)
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```
Comment (t-yanick) [inline README.md:117]: - [ ] Great job working on this milestone. Your license link is broken and given Error 404 when clicked. It is not professional to have broken links in the README. Kindly fix this.
image

File: package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "math_magicians",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
```
Comment (t-yanick) [inline package.json]: - [ ] Great job so far. You package.json is not having the correct-react scripts and as such I cannot start your application. Get the correct react scripts. This scripts are normally generated automatically if you use the [right command](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app) to create your new react app.
Your scripts should be similar to those below:
```
"scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
```

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @ybadr99 , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1 +1,117 @@
...
+> Yousef Badr
+> 👤 **Author1**
+
+- GitHub: [@ybadr99](https://github.com/ybadr99)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/yousef-mohamed-badr/)
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
```
Comment (Olamarx) [inline README.md:97]: - [OPTIONAL] I strongly advise you to always add Future Features to your readme as it is a requirement and not optional.

**Review by Olamarx — APPROVED** [review]
Hi @ybadr99, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️
- Commit messages meaningful ✔️
- CRA usage ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**